org.texmacs.TeXmacs.yml

```
@@ -0,0 +1,116 @@
+app-id: org.texmacs.TeXmacs
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: texmacs
+rename-desktop-file: texmacs.desktop
+rename-icon: texmacs
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --persist=.TeXmacs
+  - --env=PATH=/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts
+  - --env=LD_LIBRARY_PATH=/app/texlive/lib:/app/texlive/lib/perl5/5.38.2/aarch64-linux/CORE:/app/texlive/lib/perl5/5.38.2/x86_64-linux/CORE
+  - --env=PERL5LIB=/app/texlive/lib/perl5/5.38.2:/app/texlive/lib/perl5/site_perl/5.38.2
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    version: "23.08"
+    directory: texlive
+    autodelete: false
+  org.lyx.LyX.Plugin:
+    versions: stable;beta;test
```

```suggestion
    versions: stable
```

This should be only ONE. Or there is actually no point having versions.